components/config/opts.rs

```diff
@@ -21,6 +21,9 @@ use std::sync::atomic::{AtomicBool, Ordering};
 use std::sync::{RwLock, RwLockReadGuard};
 use url::{self, Url};
 
+#[cfg(all(target_os = "windows", not(target_vendor = "uwp")))]
+extern crate os_info;
```

Comment:
This statement should not be necessary in the 2018 edition of Rust, which components/config/Cargo.toml enables.